scripts/generate_proto_stubs.sh

```diff
@@ -12,45 +14,43 @@
 PROTOBUF_VERSION=3.18.1
 MYPY_PROTOBUF_VERSION=v3.0.0
 
-set -ex
-
 if uname -a | grep Darwin; then
     # brew install coreutils wget
     PLAT=osx
 else
     PLAT=linux
 fi
-REPO_ROOT=$(realpath $(dirname "${BASH_SOURCE[0]}")/..)
-TMP_DIR=$(mktemp -d)
-PYTHON_PROTOBUF_FILENAME=protobuf-python-${PROTOBUF_VERSION}.zip
-PROTOC_FILENAME=protoc-${PROTOBUF_VERSION}-${PLAT}-x86_64.zip
-PROTOC_URL=https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/${PROTOC_FILENAME}
-PYTHON_PROTOBUF_URL=https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/${PYTHON_PROTOBUF_FILENAME}
+REPO_ROOT="$(realpath "$(dirname "${BASH_SOURCE[0]}")"/..)"
```

Comment:
The quote nesting looks wrong.

Reply:
Why? 🤔 
1. `"${BASH_SOURCE[0]}"`
2. `"$(dirname "${BASH_SOURCE[0]}")"`
3. `"$(realpath "$(dirname "${BASH_SOURCE[0]}")"/..)"`